cinema/views.py

```diff
@@ -1 +1,64 @@
-# write views here
+from rest_framework import viewsets
+from django.db.models import QuerySet
+
+from cinema.models import Movie, MovieSession, CinemaHall, Actor, Genre
+from cinema.serializers import (
+    MovieSerializer,
+    MovieListSerializer,
+    MovieRetireveSerializer,
+    MovieSessionSerializer,
+    MovieSessionListSerializer,
+    MovieSessionRetireveSerializer,
+    CinemaHallSerializer,
+    ActorSerializer,
+    GenreSerializer,
+)
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+
+    def get_serializer_class(self) -> MovieSerializer:
```

Comment:
Please, don't forget about type annotation in class methods. It is not a critical issue, but it is good practice to have it in your code. It allows other developers and tools to know what type of value your function is expected to return. For example, in this case, the `get_serializer_class` method should have a return type of `Type[serializers.BaseSerializer]`.